src/server/scripts/Spells/spell_paladin.cpp

```diff
@@ -334,6 +335,28 @@ class spell_pal_blessing_of_faith : public SpellScriptLoader
         }
 };
 
+// 115750 - Blinding Light
+class spell_pal_blinding_light : public SpellScript
+{
+    PrepareSpellScript(spell_pal_blinding_light);
+
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo({ SPELL_PALADIN_BLINDING_LIGHT_EFFECT });
+    }
+
+    void HandleDummy(SpellEffIndex /*effIndex*/)
+    {
+        if (Unit* target = GetHitUnit())
+            GetCaster()->CastSpell(target, SPELL_PALADIN_BLINDING_LIGHT_EFFECT, true);
```

Comment:
what happens if the effect breaks on damage? should this aura still remains on target?

Reply:
no it removes auto as i saw on creature

Reply:
i did not test as i dont have good creatures to do on or player to test with

Reply:
forget what i said